x/authz/codec/cdc.go

```diff
@@ -2,17 +2,20 @@ package codec
 
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 var (
 	Amino     = codec.NewLegacyAmino()
 	ModuleCdc = codec.NewAminoCodec(Amino)
+	GlobalCdc *codec.ProtoCodec
```

Comment:
You can inline this as:

`ProtoCdc = codec.NewProtoCodec(types.NewInterfaceRegistry())`